win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -800,11 +814,12 @@ typedef struct CHAOS_TEST_DATA_TAG
 {
     volatile LONG64 expected_call_count;
     volatile LONG64 executed_work_functions;
-    volatile LONG64 executed_timer_functions;
+    volatile LONG executed_timer_functions;
```

Comment:
> LONG executed_t [](http://example.com/codeflow?start=12&length=16)

I think we could keep this 64 bit. It is hard but not impossible to fire 4B timers. But it will be absolutely impossible with current CPUs to fire as many times to exhaust 64 bit.

Reply:
changed back to 64